FirebaseDynamicLinks/Sources/FIRDLDefaultRetrievalProcessV2.m

```diff
@@ -296,6 +296,32 @@ - (nullable NSURL *)uniqueMatchLinkToCheck {
   return nil;
 }
 
+- (NSString *)retrievePasteboardContents {
+  __block NSString *pasteboardContents;
+
+  if (@available(iOS 14.0, *)) {
+    dispatch_group_t pasteboardGroup = dispatch_group_create();
+    dispatch_group_enter(pasteboardGroup);
+
+    NSSet<UIPasteboardDetectionPattern> *detectionPatterns =
+        [NSSet setWithArray:@[ UIPasteboardDetectionPatternProbableWebURL ]];
+    [[UIPasteboard generalPasteboard]
+        detectPatternsForPatterns:detectionPatterns
+                completionHandler:^(NSSet<UIPasteboardDetectionPattern> *patternSet,
+                                    NSError *error) {
+                  if ([patternSet containsObject:UIPasteboardDetectionPatternProbableWebURL]) {
+                    pasteboardContents = [UIPasteboard generalPasteboard].string;
+                  }
+                  dispatch_group_leave(pasteboardGroup);
+                }];
+
+  } else {
+    pasteboardContents = [UIPasteboard generalPasteboard].string;
```

Comment:
This line alone will trigger the notification to be displayed!

Reply:
> This line alone will trigger the notification to be displayed!

Looks good to me:
```swift
if (@available(iOS 14.0, *)) {
    // iOS 14+
    ...
} else {
    // iOS 13 and below - no pasteboard notification there 🙈 
    pasteboardContents = [UIPasteboard generalPasteboard].string;
}
```

Reply:
the line 306 will be executed only on iOS < 10 therefore no popup. 

Still the line 303 will trigger a popup, but I guess the point was to just "reduce" the popups  (check the Changelog